pkg/storaged/utils.js

```diff
@@ -891,15 +891,18 @@
         // HACK: get_active_usage is used for mounting and formatting so we use the absence of the subvol argument
         // to figure out that we want to format this device.
         // This is separate from the if's below as we also always have to umount the filesystem.
-        if (btrfs_volume && !subvol) {
+
+        // We allow a btrfs volume with one device to be formatted as this
+        // looks the most like a normal filesystem use case.
+        if (btrfs_volume && btrfs_volume.data.num_devices !== 1 && !subvol) {
             usage.push({
                 level,
                 usage: 'btrfs-device',
                 block,
                 btrfs_volume,
                 location: btrfs_volume.data.label || btrfs_volume.data.uuid,
                 actions: get_actions(_("remove from btrfs volume")),
-                blocking: true
+                blocking: true,
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19880-20240130-123304-85339f80-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)